aiosmtpd/docs/smtp.rst

```diff
@@ -103,60 +103,62 @@ override to provide additional responses.
 SMTP API
 ========
 
-.. class:: SMTP(handler, *, data_size_limit=33554432, enable_SMTPUTF8=False, decode_data=False, hostname=None, ident=None, tls_context=None, require_starttls=False, timeout=300, auth_required=False, auth_require_tls=True, auth_exclude_mechanism=None, auth_callback=None, loop=None)
+.. class:: SMTP(handler, *, data_size_limit=33554432, enable_SMTPUTF8=False, \
+   decode_data=False, hostname=None, ident=None, tls_context=None, require_starttls=False, \
+   timeout=300, auth_required=False, auth_require_tls=True, auth_exclude_mechanism=None, \
+   auth_callback=None, loop=None)
```

Comment:
After some experimentation, I found out how to split an overly long line into several: Split using backslash, then indent.